kategory/src/main/kotlin/kategory/instances/IdInstances.kt

```diff
@@ -41,3 +41,14 @@ interface IdInstances :
             fa.ev().value
 
 }
+
+object IdSemigroupK : SemigroupK<Id.F>, GlobalInstance<SemigroupK<Id.F>>() {
+    override fun <A> combineK(x: HK<Id.F, A>, y: HK<Id.F, A>): HK<Id.F, A> = y
+}
+
+class IdMonoidK(val a: Any) : MonoidK<Id.F>, GlobalInstance<MonoidK<Id.F>>() {
```

Comment:
My intuition says that this should be an instance of Monoid for the wrapped type. That way you can use IntMonoid, or ListMonoid or similar.